steward/src/cork.rs

```diff
@@ -94,6 +105,140 @@ impl steward::contract_call_server::ContractCall for CorkHandler {
     }
 }
 
+pub struct DirectCorkHandler;
```

Comment:
There needs to be a CLI command similar to `steward/src/comands/start.rs` that uses the `DirectCorkHandler` instead of the `CorkHandler` so the server can be run in test mode.